src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -13254,32 +13238,34 @@ namespace OutputReportTabular {
                 for (int jSurf = Zone(zoneIndex).SurfaceFirst; jSurf <= Zone(zoneIndex).SurfaceLast; ++jSurf) {
                     if (!Surface(jSurf).HeatTransSurf) continue; // Skip non-heat transfer surfaces
 
-                    // determine for each timestep the amount of radiant heat for each end use absorbed in each surface
-                    Real64 QRadThermInAbsMult =
-                        TMULTseq(desDaySelected, kTimeStep, radEnclosureNum) * ITABSFseq(desDaySelected, kTimeStep, jSurf) * Surface(jSurf).Area;
-                    peopleRadIntoSurf(kTimeStep) = peopleRadSeq(desDaySelected, kTimeStep, zoneIndex) * QRadThermInAbsMult;
-                    equipRadIntoSurf(kTimeStep) = equipRadSeq(desDaySelected, kTimeStep, zoneIndex) * QRadThermInAbsMult;
-                    hvacLossRadIntoSurf(kTimeStep) = hvacLossRadSeq(desDaySelected, kTimeStep, zoneIndex) * QRadThermInAbsMult;
-                    powerGenRadIntoSurf(kTimeStep) = powerGenRadSeq(desDaySelected, kTimeStep, zoneIndex) * QRadThermInAbsMult;
-                    lightLWRadIntoSurf(kTimeStep) = lightLWRadSeq(desDaySelected, kTimeStep, zoneIndex) * QRadThermInAbsMult;
-                    // for each time step, step back through time and apply decay curve
+                    // for each time step, step back through time and apply decay curve to radiant heat for each end use absorbed in each surface
                     Real64 peopleConvFromSurf = 0.0;
                     Real64 equipConvFromSurf = 0.0;
                     Real64 hvacLossConvFromSurf = 0.0;
                     Real64 powerGenConvFromSurf = 0.0;
                     Real64 lightLWConvFromSurf = 0.0;
                     Real64 lightSWConvFromSurf = 0.0;
                     Real64 feneSolarConvFromSurf = 0.0;
-                    for (int mStepBack = 1; mStepBack <= kTimeStep; ++mStepBack) {
```

Comment:
Previously, the history terms only accumulated from midnight to the current `kTimeStep`, so an early morning peak would only include a few hours of history.

Reply:
So now it is wrapping around the same design day just looking at the end of the day to impact the beginning of the day. I'm not sure that makes sense to me.

Reply:
I was just thinking about this kTimeStep loop and mStepBack internal loop. What it is summing, for each TimeStep in the peak day, is the radiant loads for the previous 24 hours with the decay curve applied yielding this TimeStep's convection load. If you do this over the day using kTimeStep, regardless of where you start the sum, aren't you summing the same data again and getting the same answer? Unless the decayCurve is shifting each time which would affect the sum. This is confusing.

Reply:
Yeah, it does look like the decayCurve is shifting each time and yielding the previous 24 hours from the TimeStep. In this case the decay curve needs to be a full 24 hours.